deploy/templates/nstemplatetiers/basedeactivationdisabled/cluster.yaml

```
@@ -125,6 +125,20 @@ objects:
       annotations:
         openshift.io/requester: ${USERNAME}
       labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-rhoas
+  spec:
+    quota:
+      hard:
+        count/cloudservicesrequests.rhoas.redhat.com: "2"
+        count/kafkaconnections.rhoas.redhat.com: "6"
```

```suggestion
        count/kafkaconnections.rhoas.redhat.com: "5"
```

Since we want to prevent from summit user causing rate limiting issue we can reduce number of Kafka connection to 4